openbb_terminal/mutual_funds/mutual_fund_controller.py

```diff
@@ -47,12 +45,15 @@ class FundController(BaseController):
         "plot",
         "sector",
         "equity",
-        "alswe",
-        "infoswe",
-        "forecast",
+        # "alswe",
+        # "infoswe",
```

Comment:
These two can be left in.  But only showing when sweden is selected (like it was)